backend/mora/common.py

```diff
@@ -50,6 +51,112 @@ def get(self, objid):
         pass
 
 
+class RequestType(enum.Enum):
+    CREATE = 0
+    EDIT = 1
+
+
+# Handlers is populated by each individual active RequestHandler
+HANDLERS = {}
+
+
+def register_request_handler(name):
```

Comment:
`register` is redundant — actually, how about `handler_for()`?

Reply:
Well, I wanted it to be obvious that adding the decorator would actively register the decorated handler with the system.

Reply:
It's just that this chunk of code feels somewhat repetitive:

```python
@common.register_request_handler('address')
 class AddressRequestHandler(common.RequestHandler):
```

Anyway, it's definitely a minor issue, so let's not hold up this change on it. Plus, you wrote the code, after all!